README.md

```diff
@@ -6,33 +6,87 @@ At a high level the key thing you will contribute is a `Dockerfile`, which will
+
+As you're working on your submission `Dockerfile` you'll want to test it out locally to make sure your contribution works as expected before you submit it.
+
+HELM makes it easy to add new evaluation datasets by just adding another line in a config file so make sure to experiment with the different datasets they have available and feel free to contribute your own.
+
+To learn more about how to test your submission with HELM, please follow the instructions [here](helm.md).
+
+## Finetune
 
-### OpenAPI Specification
+It's likely that an untuned base model won't give you satisfactory results, in that case you might find it helpful to do some additional finetuning. There are many frameworks to do this but we've created 2 sample submissions to do so
+1. [lit-gpt](/sample-submissions/lit-gpt/)
+2. [llama-recipes](/sample-submissions/llama_recipes/)
+
+
+### Your own submission template
+
+Note that we've offered 2 sample submissions, our evaluation infrastructure is generic and only assumes an HTTP client so you can use a finetuning framework in python like the ones we've suggested but also any non based python framework you like using.
 
 The `openapi.json` file in this repository contains the OpenAPI specification for the Competition API. Competitors can use this specification to understand the API endpoints, request and response structures, and overall requirements for interacting with the competition platform.
 
 The OpenAPI specification provides a standardized way to describe the API, making it easier for competitors to develop their own solutions and integrate them seamlessly with the competition infrastructure.
 
 
-## HELM
+## Discord Leaderboard
 
-Every submission will be tested against [HELM](https://crfm.stanford.edu/helm/latest/) which is a standard suite to evaluate LLMs on a broad set of datasets. This competition will leverage HELM for its evaluation infrastructure. The organizers will leverage standard STEM tasks from HELM although we will keep the exact set a secret and in addition we'll be including some heldout tasks that are presently not in HELM.
+The [Lightning.ai](https://lightning.ai/) has built a Discord based for us. You can find it on discord by its name `evalbot#4372`.
 
-As you're working on your submission `Dockerfile` you'll want to test it out locally to make sure your contribution works as expected before you submit it.
+You can interact with it by DM'ing it with a zipped file of your sample submission and message it to either `eval A100` or `eval 4090`. More details on the bot are [here](https://discord.com/channels/1124130156336922665/1124134272631054447/1151718598818156645)
 
-To learn more about how to test your submission with HELM, please follow the instructions [here](helm.md).
+Once you make a submission the bot will inform you whether your submission failed or succeeded and after a few hours will publicly post your results. Your submission will remain private to other competitors.
```

Comment:
Maybe add how long one should expect evaluation to take (especially if you're queued up). And also  how important it is to first verify that the evaluation succeeds locally before submitting.